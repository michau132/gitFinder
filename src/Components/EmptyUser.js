import React from 'react';
import { Typography } from '@material-ui/core';

const EmptyUser = () => (
  <Typography variant="h5">
    Here will be a profile from Github. Enter above a Github login to see more.
  </Typography>
);

export default EmptyUser;
