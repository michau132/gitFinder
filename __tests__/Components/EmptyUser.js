import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import EmptyUser from '../../src/Components/EmptyUser';

describe('testing EmptyUser Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<EmptyUser />);
  });

  test('renders without crashing', () => {
    expect(wrapper).toBeDefined();
  });

  test('should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
