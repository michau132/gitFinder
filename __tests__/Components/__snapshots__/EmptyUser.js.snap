// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing EmptyUser Component should match snapshot 1`] = `
<WithStyles(Typography)
  variant="h5"
>
  Here will be a profile from Github. Enter above a Github login to see more.
</WithStyles(Typography)>
`;
